lib/codecs/tests/bin/generate-avro-fixtures.rs

```diff
@@ -142,23 +142,6 @@ fn generate_avro_test_case_string() {
     generate_test_case(schema, value, "string");
 }
 
-fn generate_avro_test_case_fixed() {
```

Comment:
Let's not remove these since they might help future contributors. You can just tag them with `#[allow(unused)]`

Reply:
Sounds good, so keep the functions but not use them in the function that generates the files?

Reply:
Yes, and use `#[allow(unused)]` to make clippy happy.